reana_cluster/configurations/reana-cluster.yaml

```diff
@@ -16,26 +16,26 @@ cluster:
 components:
   reana-workflow-controller:
     type: "docker"
-    image: "reanahub/reana-workflow-controller:v0.4.0-52-g31ece9b"
+    image: "reanahub/reana-workflow-controller:0.5.0"
     environment:
       - <<: *db_base_config
       - ORGANIZATIONS: "default,alice,atlas,cms,lhcb"
-      - REANA_WORKFLOW_ENGINE_IMAGE_CWL: "reanahub/reana-workflow-engine-cwl:v0.4.0-20-g065377c"
-      - REANA_WORKFLOW_ENGINE_IMAGE_YADAGE: "reanahub/reana-workflow-engine-yadage:v0.4.0-15-g4b198e0"
-      - REANA_WORKFLOW_ENGINE_IMAGE_SERIAL: "reanahub/reana-workflow-engine-serial:v0.4.0-18-g6e7b2b2"
+      - REANA_WORKFLOW_ENGINE_IMAGE_CWL: "reanahub/reana-workflow-engine-cwl:0.5.0"
+      - REANA_WORKFLOW_ENGINE_IMAGE_YADAGE: "reanahub/reana-workflow-engine-yadage:0.5.0"
+      - REANA_WORKFLOW_ENGINE_IMAGE_SERIAL: "reanahub/reana-workflow-engine-serial:0.5.0"
 
   reana-job-controller:
     type: "docker"
-    image: "reanahub/reana-job-controller:0.4.0-17-g9eec312"
+    image: "reanahub/reana-job-controller:0.5.0"
```

Comment:
This needs to be updated to `0.5.1` see https://hub.docker.com/r/reanahub/reana-job-controller/tags